src/components/RichTextPopup/RichTextPopup.js

```diff
@@ -1,29 +1,31 @@
-import React, { useState, useEffect, useRef } from 'react';
-import Draggable from 'react-draggable';
-import classNames from 'classnames';
-import { useSelector, useDispatch, shallowEqual } from 'react-redux';
-
-import Element from 'components/Element';
-import ColorPalette from 'components/ColorPalette';
-import Button from 'components/Button';
-import HorizontalDivider from 'components/HorizontalDivider';
-import { isMobile } from 'helpers/device';
-import core from 'core';
-import getRichTextPopupPosition from 'helpers/getRichTextPopupPosition';
-import actions from 'actions';
-import selectors from 'selectors';
-
-import './RichTextPopup.scss';
+import React, { useState, useEffect, useRef } from "react";
+import Draggable from "react-draggable";
+import classNames from "classnames";
+import { useSelector, useDispatch, shallowEqual } from "react-redux";
+
+import Element from "components/Element";
+import ColorPalette from "components/ColorPalette";
+import Button from "components/Button";
+import HorizontalDivider from "components/HorizontalDivider";
+import { isMobile } from "helpers/device";
+import core from "core";
+import getRichTextPopupPosition from "helpers/getRichTextPopupPosition";
+import MathSymbolsPicker from "../MathSymbolsPicker";
+import actions from "actions";
+import selectors from "selectors";
+
+import "./RichTextPopup.scss";

```

Comment:
nit: string should use `''` instead of `""` for consistency

Reply:
I ran VS code file auto-format on this file, convert the double quote to single quote marks